src/tiles/clickable-tile.component.ts

```diff
@@ -34,6 +39,7 @@ import { Router } from "@angular/router";
 		(click)="navigate($event)"
 		[attr.href]="disabled ? null : href"
 		[attr.target]="target"
+		[attr.rel]="rel"
```

Comment:
Define `rel` as an input property and use something like `[attr.rel]="rel ? rel : null"`

Reply:
Sure, have added this
I had thought that having the rel attribute for all links would be helpful for usage when rel is not mentioned